test/eoftests/eoftests.hpp

```diff
@@ -0,0 +1,40 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2022 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+
+#include <evmc/evmc.hpp>
+#include <evmone/eof.hpp>
+#include <nlohmann/json.hpp>
+#include <filesystem>
+
+namespace fs = std::filesystem;
+namespace json = nlohmann;
+
+namespace evmone::test
+{
+struct EOFTests
```

Comment:
This struct represents what test framework calls a test (with multiple cases inside), so better to rename
```suggestion
struct EOFValidationTest
```

Reply:
Also would be good rename files and directory to `eoftest` for consistency with `statetest`